python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -648,8 +648,6 @@ public void visitQualifiedExpression(QualifiedExpression qualifiedExpression) {
 
     @Override
     public void visitDecorator(Decorator decorator) {
-      Name nameTree = decorator.name().names().get(0);
```

Comment:
I think that we can remove this lines because decorator name is now a variable. I think it would be nice to add a test case inside `NameImpl` to assert that

Reply:
As discussed, I added the test in `SymbolTableBuilderTest` instead.